Python/pywarpx/callbacks.py

```diff
@@ -256,7 +256,6 @@ def callfuncsinlist(self,*args,**kw):
 _c_afterdeposition = _CALLBACK_FUNC_0(_afterdeposition)
 libwarpx.warpx_set_callback_py_afterdeposition(_c_afterdeposition)
 _c_fieldsolver = _CALLBACK_FUNC_0(_fieldsolver)
-libwarpx.warpx_set_callback_py_fieldsolver(_c_fieldsolver)
```

Comment:
Why did this line disappear?

Reply:
If it is there `if (warpx_py_fieldsolver)` in `WarpXEvolve.cpp` evaluates to True even if a field solver is not installed since `warpx_py_fieldsolver` is no longer None. This is a bugfix in the implementation.